### DISCUSSION TITLE
[Feature] "skip/next" option

### DISCUSSION THREAD

**Discussion by blublub** [immich-app/immich#10402] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Simple feature request:

The current check duplicate option/workflow requires the user to make a decision in order to work through the duplicates - sometimes its difficult to make a decision but ppl still want to move on with the next duplicate - so a "next" option would be usefull I think

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alexkutsan** [comment]
Yeah! I just came here to ask for the same feature

**Comment by realtek2017** [comment]
I just came here for this too! I don't want to have to decide all of them upfront. Perhaps is want to go through them first. This makes it hard to use this tool

**Comment by ireun** [comment]
Until this feature is implemented the 'duplication' finder is useless for me, sadly

**Comment by xCJPECKOVERx** [comment]
Done #20880

**Comment by ireun** [comment]
Huge thank you! 

Only one small regression - when zoomed in to a photo - left/right arrows should move between photos A/B not through duplicates prev/next

Edit: Already reported in https://github.com/immich-app/immich/issues/21160
